zookeeper-api/src/main/java/org/apache/helix/zookeeper/impl/client/SharedZkClient.java

```diff
@@ -189,6 +189,36 @@ public void createPersistent(String path, Object data, List<ACL> acl) {
     _innerSharedZkClient.createPersistent(path, data, acl);
   }
 
+  @Override
+  public void createPersistentWithTTL(String path, long ttl) {
```

Comment:
This comment applies to all the code below:
Why don't we call the same method with more parameters, instead we repeat the code?
Taking this as an example, can we do `createPersistentWithTTL(path, false, ttl)` here?

Reply:
Thanks for the suggestion, yes that makes sense. I will make that change.